solutions/standard/catalogValidationValues.json.template

```diff
@@ -1,5 +1,4 @@
 {
   "ibmcloud_api_key": $VALIDATION_APIKEY,
-  "sm_service_plan": "trial",
   "cert_common_name": $DNS_CLOUD_DOMAIN
```

Comment:
Isn't there more required variables that are needed here? Such as an existing secrets manager instance?

Reply:
@akocbek this comment is still open